src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -116,13 +111,42 @@ private static bool IsNamespaceName(NameSyntax name)
             }
         }
 
+        private void TryClassifyStaticSymbol(
+            ISymbol symbol,
+            Text.TextSpan span,
+            ArrayBuilder<ClassifiedSpan> result)
+        {
+            if (symbol?.IsStatic == true
+                && (!symbol.IsKind(SymbolKind.Field) || symbol.ContainingType?.IsEnumType() == false) // TODO: Since Enum members are always static is it useful to classify them as static?
+                && !symbol.IsKind(SymbolKind.Namespace)) // TODO: Since Namespace are always static is it useful to classify them as static?
+            {
+                result.Add(new ClassifiedSpan(span, ClassificationTypeNames.StaticSymbol));
+            }
+        }
+
         private bool TryClassifySymbol(
             NameSyntax name,
             ISymbol symbol,
             SemanticModel semanticModel,
             CancellationToken cancellationToken,
             out ClassifiedSpan classifiedSpan)
         {
+            if (symbol is INamespaceSymbol namespaceSymbol
+                && name is IdentifierNameSyntax identifierNameSyntax)
+            {
+                // Continue to classify the global:: namespace as a keyword
+                var isGlobalNamespace = namespaceSymbol.IsGlobalNamespace 
+                    && identifierNameSyntax.Identifier.IsKind(SyntaxKind.GlobalKeyword);
```

Comment:
this feels weird.  Doesn't 'keyword' have the highest classification priority anyways?  Why is this necessary?

Reply:
Semantic classifications override syntactic classifications so we opt to not classify it and allow it to take the syntactic 'keyword' classification.